### PR TITLE
 add new python file to add 2 numbers

### PR DESCRIPTION
**Pull request by YuweAEC**

### CODE REVIEW

File: simple-operations.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+a=5
+b=10
+sum=a+b
+sub=a-b
+devide=a/b
+multiply=a*b
+print("sum of a and b is",sum)
+print("sub of a and b is",sub)
+print("devide of a and b is",devide)
+print("multiply of a and b is",multiply)
```
Comment (gauravsinghaec) [inline simple-operations.py]: multiply of a & b, is this correct English?

**Review by gauravsinghaec — COMMENTED** [review]
Please correct the print statement English

File: simple-operations.py (hunk 1 above)
Comment (gauravsinghaec) [reply simple-operations.py]: Where are the screenshots?

Comment (YuweAEC) [reply simple-operations.py]: Screenshot done

Comment (YuweAEC) [reply simple-operations.py]: english corrected

**Review by YuweAEC — COMMENTED** [review]
Statement corrected

**Pull request merged**